app/controllers/api/v1/tracks_controller.rb

```diff
@@ -5,7 +5,7 @@ class TracksController < Api::BaseController
       respond_to :json
 
       def index
-        @conference ? (tracks = @conference.tracks) : (tracks = Track.all)
+        @conference ? (tracks = @conference.program.tracks) : (tracks = Track.all)
```

Comment:
Okay! Ever heard from the [boyscout rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule)? ;)